README.md

```diff
@@ -550,6 +550,26 @@ sent via this stream.
 See more details about these APIs in
 [grpc/\_client.py](src/python/library/tritonclient/grpc/_client.py).
 
+For gRPC AsyncIO requests, an AsyncIO task wrapping an `infer()` coroutine can
+be safely cancelled.
+
+```python
+  infer_task = asyncio.create_task(aio_client.infer())
+  await something_else
```

Comment:
is this `await something_else` necessary/useful to the example?

Reply:
It is not necessary to demonstrate cancellation. It is removed for simplicity.

A side note: we are assuming our users should know AsyncIO is single threaded. Without `await` between creating the task and cancelling the task, there is no context switching, so the task is cancelled without given a chance to run. Thus, the task should not have been created in the first place, in the real world.